module/VuFind/src/VuFind/Role/PermissionProvider/PluginManager.php

```diff
@@ -46,6 +46,7 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
      * @var array
      */
     protected $aliases = [
+        'insecurecookie' => InsecureCookie::class,
```

Comment:
This should probably be `insecureCookie` rather than `insecurecookie` for consistency.

Reply:
thanks @demiankatz I've implemented that in my latest follow up!